docs/user-guide/timeouts/timeouts-main.md

```diff
@@ -0,0 +1,331 @@
+# Timeout Prevention
+
+Timeout prevention approaches fall into these categories.
+1. Changing tool settings
+2. Changing specs
+3. Changing source code
+
+Checking tool settings is least invasive and easy to do, so it is preferred.
+However, there are cases when parts of the input code that are very tricky need
+to be worked around. Sometimes a combination of approaches is needed to resolve
+a timeout.
+
+
+In the following we will discuss some concrete approaches to timeout prevention.
+This collection will be extended over time based on user's experiences and tool
+improvements.
+
+```{note}
+The old documentation has a section on
+{doc}`troubleshooting </docs/confluence/perplexed>` that addresses timeouts, 
+which might complement the information given here.  
+There is also some helpful information in the section on
+{ref}`summarization <old-summary-example>`.
+Some of the information in these references is out of date.
+```
+
+## Prover Settings
+
+In this subsection we list some option combinations that have helped preventing
+timeouts in the past. We group the options by timeout causes they are most
+relevant for. 
+
+### Dealing With a High Path Count
+
+The Certora Prover internally divides each verification condition into smaller
+subproblems and attempts to solve them separately. This technique is called
+*control flow splitting*. For a more detailed explanation of how control flow
+splitting works, see [this page](control-flow-splitting).
+
+We list a few option combinations that can help in various settings. There is
+a tradeoff between spending time in different places: The Prover can either try
+to spend much time at a low splitting level in the hope that no further
+splitting will be needed, or it can split quickly in the hope that the
+subproblems will be much easier to solve. The first variant ("lazy splitting")
+is weak when the shallow splits are too hard, and time spent on them is wasted.
+The second variant ("eager splitting") is weak when we end up with too many
+subproblems; the number of splits is worst-case exponential in the splitting
+depth.
+
+When the relevant source code is very large, the shallow splits have a chance of
+being too large for the solvers, thus eager splitting might help.
+
+```
+--prover_args "-smt_initialSplitDepth 5 -depth 15"
+```
+
+When there are very many subproblems that are of medium difficulty there is a
+chance that the Prover has to split too often (not being able to "close" any
+sub-splits). Then, a lazier splitting strategy could help. We achieve lazier
+splitting by giving the solver more time to find a solution before we split a
+problem.
+
+```
+--prover_args "-mediumTimeout 30 -depth 5"
+```
+
+It can also help to have splitting run in parallel; this goes together with
+choosing a smaller solver portfolio, so the available cores can be used for
+different splits, rather than different solvers.
+
+```
+--prover_args "-smt_splitParallel true -solvers [z3:def,cvc5:def]"
+```
+
```

Comment:
`--prover_args "-dontStopAtFirstSplitTimeout true -depth 15 -mediumTimeout 5" --smt_timeout 10` is a good config for finding violations. Especially, `-dontStopAtFirstSplitTimeout true` should be described somewhere here I think.

Reply:
ah, great, adding the option here...

more on control flow splitting will come in #164 , I'm adding some stubs of entries in options.md now, too, but I don't want to pull too much over, since this PR is big enough already